app/src/main/java/swati4star/createpdf/fragment/SplitFilesFragment.java

```diff
@@ -120,6 +119,9 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) throw
 
     @OnClick(R.id.splitFiles)
     public void parse() {
+        // hide the soft keyboard on click of this button
+        hideKeyboard(mActivity);
```

Comment:
I would suggest removing this comment. The functionality is pretty clear just by the name of the method called. Comment is unnecessary.

Reply:
ok